vuepress/docs/next/docs/reference/glossary.md

```diff
@@ -38,37 +12,37 @@
 | Entando Web Content Management System (WCMS) | Entando's web content management system |
 | Local Hub | The local component repository in an Entando App Builder |
 | Entando Platform Capability (EPC) | A packaged capability that adds functionality to the platform and/or additional UX controls to the App Builder |
-| Entando Portal UI | 
-| Kubernetes integration service | Provides an abstraction layer between Entando microservices and the APIs exposed by a Kubernetes cluster |
+| Entando UX Fragment | An HTML block containing Freemarker tags that allow content to be rendered dynamically according to context |
+| Entando Widget | A UI element that can be dragged and dropped onto a page, e.g. a snippet of HTML code added from the browser or a micro frontend, which is a specialized widget |
 
 ## General Technical Terms
 
 | Term |  Description
 |:--|:--
-| backend for frontend (BFF) | A software architecture pattern that connects the frontend of an application with the backend microservices |
+| application composition platform (ACP) | Any development platform that supports a catalog of packaged, composable components that can be assembled into applications, where changes to individual components proliferate to the assembly |
+| backend for frontend (BFF) | A type of microservice dedicated to a frontend that may also act as a facade to other enterprise microservices |
 | component | An application building block such as a page template, content template, UX Fragment, widget, micro frontend, microservice, etc. |
 | component collection | A packaged set of single components that are in some way functionally unrelated |
-| content | The information/data contained within a component |
-| content type | The structure of specific content, defined by a set of content attributes |
-| content attribute | The basic data that define a content type |
-| content template | The style or layout of a content type where a single content type can have multiple content models defining different ways to render the same content |
-| digital asset | Images, documents, and other media files supported in a specific format |
 | low-code | A software development approach that leverages visual interfaces with basic logic and drag-and-drop capabilities to build applications and processes |
 | micro frontend (MFE) | A web development approach that implements an architecture characterized by a composition of compact frontend applications |
 | microservice | A software development technique characterized by an architecture of loosely coupled, granular services that employ lightweight protocols |
 | module | A basic element partitioned into a system of building blocks (e.g. PBCs, components, bundles, templates) that are independent, reusable, and can easily be configured into complex and useful structures |
 | no-code | A programming platform on which applications are built exclusively using a visual development interface to drag and drop software components |
```

Comment:
i cross-referenced definitions when googling no/low/pro-code. no-code is a platform because it's like using a GUI for everything. like in engineering there's this ancient but popular "language" called labview where you drag and drop circuit elements and the program writes the code based off of your wirings. this is a development approach, but more than that it's a platform. it's a pseudo-physical thing, self-contained, with rules and representations and all the rest.

Reply:
but no-code is never used on it's own. It's always used as an adjective, in conjunction with nouns like no-code platforms, no-code apps, etc. No one uses it on its own so it's strange to describe it as a noun.

Reply:
yes, it is. and it's defined repeatedly as a platform by reputable sources.